x-pack/filebeat/module/o365/audit/config/pipeline.js

```diff
@@ -754,12 +754,14 @@ function AuditProcessor(tenant_names, debug) {
             {from: "client.address", to: "client.ip", type: "ip"},
             {from: "server.address", to: "server.ip", type: "ip"},
             {from: "client._ip",     to: "client.ip", type: "ip"},
+            {from: "client._port",   to: "client.port", type: "long"},
```

Comment:
I don't think we need to store port in a temp var.  I think if we just use "from" without a "to", it will update in place.

Reply:
The reason is so that if the conversion fails we aren't left with data in `client.port` that will fail to index.